MAUI/Radial-Menu/Getting-Started.md

```diff
@@ -9,19 +9,33 @@ documentation: ug
 
 # Getting Started with .NET MAUI RadialMenu Control (SfRadialMenu)
 
-This section provides a quick overview of how to get started with the [.NET MAUI RadialMenu control (SfRadialMenu)](https://www.syncfusion.com/maui-controls/maui-radial-menu) for .NET MAUI and a walk-through to configure the .NET MAUI RadialMenu control in a real-time scenario.
+This section guides you through setting up and configuring a [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu) in your .NET MAUI application. Follow the steps below to add a basic [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu) to your project.
 
-## Creating an Application using the .NET MAUI RadialMenu Control
 
-1. Create a new .NET MAUI application in Visual Studio.
 
-2. Syncfusion .NET MAUI components are available in [nuget.org](https://www.nuget.org/). To add the [.NET MAUI RadialMenu]() to your project, open the NuGet package manager in Visual Studio, search for `Syncfusion.Maui.RadialMenu`, and then install it.
+## Prerequisites
+Before proceeding, ensure the following are in place:
+1. Install .NET 7 SDK or later.
 
-3. To initialize the control, import the control namespace `Syncfusion.Maui.RadialMenu` in XAML or C# code.
+2. Set up a .NET MAUI environment with Visual Studio 2022 (v17.3 or later) or VS Code. For VS Code users, ensure that the .NET MAUI workload is installed and configured as per the instructions provided [here](https://learn.microsoft.com/en-us/dotnet/maui/get-started/installation?view=net-maui-8.0&tabs=visual-studio-code).
 
-4. Initialize [SfRadialMenu](https://www.syncfusion.com/maui-controls/maui-radial-menu).
+To get start quickly with our [.NET MAUI RadialMenu control](https://www.syncfusion.com/maui-controls/maui-radial-menu), you can check the below video.
 
-## Register the handler
+{% youtube "https://www.youtube.com/watch?v=VxufXFzSh0M" %}
+
+## Step 1: Create a New MAUI Project
+
+1. Open Visual Studio or VS Code.
+2. Go to File > New > Project and choose the .NET MAUI App template.
```

Comment:
Navigate to **File > New > Project,** then select the **.NET MAUI App** template.